### ISSUE TITLE
Allow switching CCP at runtime

### ISSUE THREAD

**Issue by skx** [skx/cpmulator#87]
Now that we've added a "secret" BIOS function for interacting with the emulator we've used it a bunch of times:

* Change the Ctrl-C requirements for rebooting.
* Change the output driver, at runtime.

The obvious next step is to allow switching CCP at runtime too, via `CCP.COM`.  Run that with the name of the thing you want to run and it should just work.

We'll need to overhaul the main loop which runs:

* Forever
  * Load CCP
  * Execute it

But that's not a bad thing to do anyway.  `main.go` is getting messy as it is and should be cleaned up in a separate issue.

TLDR:

* Add syscall for changing CCP.
* Add samples/CCP.{z80 COM}.
   *  Add to cpm-dist too. 
* Make it work.
